MusicTaste/MusicTaste/View Controllers/ProfileViewController.m

```diff
@@ -1,13 +1,30 @@
 #import "ProfileViewController.h"
+#import "UIImageView+AFNetworking.h"
+#import "User.h"
 
 @interface ProfileViewController () 
+//connects objects: Screenname and profileview
+@property (weak, nonatomic) IBOutlet UILabel *screen_name;
+@property (weak, nonatomic) IBOutlet UIImageView *profileView;
+
 
 @end
 
 @implementation ProfileViewController
-    
+
 - (void)viewDidLoad {
     [super viewDidLoad];
+    //Sets username
+    self.screen_name.text = [[User user] name];
+    //Gets image Url
+    NSString *URLString = [[User user] profilePicture];
+    NSString *stringWithoutNormal = [URLString stringByReplacingOccurrencesOfString:@"_normal" withString:@""];
```

Comment:
The comments "sets username" and "gets image url" are not that helpful, since that's kind of obvious from the code. It's not obvious why we need to remove `_string` though; that would have been good to comment on.

Reply:
Will do